src/components/Reports/SubscriberHistory/SubscriberHistory.js

```diff
@@ -59,269 +21,30 @@ const SubscriberHistory = ({
-                                          />
-                                        </div>
-                                      </a>
-                                    </div>
-                                  ) : (
-                                    <a
-                                      href={`${reportsUrl}/Dashboard.aspx?idCampaign=${campaign.campaignId}`}
-                                    >
-                                      {campaign.campaignName}
-                                    </a>
-                                  )}
-                                </td>
-                                <td>{campaign.campaignSubject}</td>
-                                <td>
-                                  <span
-                                    className={getDeliveryStatusCssClassName(
-                                      campaign.deliveryStatus,
-                                    )}
-                                  >
-                                    <FormattedMessage
-                                      id={
-                                        'subscriber_history.delivery_status.' +
-                                        campaign.deliveryStatus
-                                      }
-                                    />
-                                  </span>
-                                </td>
-                                <td>{campaign.clicksCount}</td>
-                              </tr>
-                            ))}
-                          </>
-                        ) : (
-                          <tr>
-                            <td>
-                              <span className="bounceIn">
-                                <FormattedMessage id="subscriber_history.empty_data" />
-                              </span>
-                            </td>
-                            <td></td>
-                            <td></td>
-                            <td></td>
-                          </tr>
-                        )}
-                      </tbody>
-                    </table>
-                  )}
-                </div>
+    <SubscriberPage
+      header={() => (
+        <header className="hero-banner report-filters">
+          <div className="dp-container">
+            <div className="dp-rowflex">
+              <div className="col-sm-12 col-md-12 col-lg-12">
+                <Breadcrumb />
+                <h2>
+                  <FormattedMessage id="subscriber_history.title" />
+                </h2>
+                <p>
+                  <FormattedMessage id="subscriber_history.description" />
+                </p>
               </div>
             </div>
+            <span className="arrow"></span>
           </div>
-        </div>
-      </section>
-    </>
+        </header>
+      )}
+      content={({ subscriber }) => (
+        <SubscriberSentCampaigns subscriber={subscriber} location={location} />
```

Comment:
It would be nice to avoid passing location. @fcoronelmakingsense do you think that it is possible reading the location directly inside _SubscriberSentCampaigns_?